kore/app/exec/Main.hs

```diff
@@ -594,21 +596,21 @@ mainWithOptions execOptions = do
             ("// Last configuration:\n" <> unparse lastConfiguration)
         throwM someException
 
-    go :: Main ExitCode
-    go
+    go :: KoreExecOptions -> Main ExitCode
```

Comment:
Because we are passing `KoreExecOptions` into this function now, can we extract it to the top level?